src/docs/sdk/performance/dynamic-sampling-context.mdx

```diff
@@ -118,7 +122,7 @@ The value of this envelope header is a JSON object with the following fields:
 - `sample_rate` (string) - The sample rate as defined by the user on the SDK. This string should always be a number between (and including) 0 and 1 in basic float notation (`0.04242`) - no funky business like exponents or anything similar. If a `tracesSampler` callback was used for the sampling decision, its result should be used for `sample_rate` instead of the `tracesSampleRate` from `SentryOptions`. In case `tracesSampler` returns `True` it should be sent as `1.0`, `False` should be sent as `0.0`.
 - `release` (string) - The release name as specified in client options`.
 - `environment` (string) - The environment name as specified in client options.
```

Comment:
So only sending this in the envelope header if `tracingOrigins` allows would mean we have to check `tracingOrigins` against the DSN or proxy to see if we're allowed to send it, correct?

Reply:
Are you referring to `user_id`? Could you elaborate what you mean here? Generally, in the context of this PR we're not including `tracingOrigins` in any way or condition. Just wanted to mention in the description that discussions around it are in the works.

Reply:
Yes, this referred to `user_id`. Once discussed we could add something to clarify whether we need to check `tracingOrigins` against DSN / proxy.

Reply:
So currently, the JS SDK's `BrowserTracing` integration does not check for the DSN (or a proxy) but it only [determines](https://github.com/getsentry/sentry-javascript/blob/cb2462b578ced62e9d22cecfd72fdebaca42b62c/packages/tracing/src/browser/request.ts#L116-L125) if the URL of the outgoing request matches a string or regex specified in `tracingOrigins`. So far, we haven't considered changing this but since`tracingOrigins` is gonna be standardized, we can certainly discuss this. I would suggest though, discussing this in a separate issue about the `tracingOrigins` spec/changes.

Reply:
For Java we have our own transport(s) for communication with Sentry so we'd have to separately add the check there if we so choose.

Reply:
`tracingOrigins` doesn't really matter when determining what is sent to Sentry. `tracingOrigins` should simply control which targets should receive a) a `sentry-trace` header b) DSC in a baggage header.

This shouldn't concern transports but rather request instrumentation. I might be wrong though since I don't know specifics on the Java SDK.